packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -166,21 +173,59 @@ class TechnologyController {
 		});
 	}
 
+	async syncronizeTerms(terms, technology, detach = false) {
```

Comment:
O que acontece se eu tentar criar/atualizar uma tecnologia com um termo que não existe? Pelo que estou entendo ele vai criar/atualizar a tecnologia mas lançar uma excessão se o termo não existir.

Acho que vamos ter que fazer isso em uma transação para facilitar as coisas, pois se algum erro for lançado fazemos um fallback na transação, o que acha?

Reply:
Tem razão! Não tinha atentado para isso. Até testei aqui só para confirmar e, de fato, isso que acontece. Para eu lembrar depois:
- Colocar cadastro e atualização em uma transação;
- Atualizar o teste de cadastro e atualização de tecnologia para garantir que a tecnologia não é criada (atualmente, só verificamos o status 400 e a msg de retorno)